DependencyInjection/Compiler/ReplaceUserCheckerCompilerPass.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Modera\SecurityBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author    Konstantin Myakshin <koc-dp@yandex.ru>
```

Comment:
Probably it wouldn't hurt to give some background regarding why this class exists and why it can be removed when MF is upgraded to Symfony 2.8.